src/Mod/TechDraw/Gui/QGISVGTemplate.cpp

```diff
@@ -258,6 +260,12 @@ void QGISVGTemplate::clearClickHandles()
 
 void QGISVGTemplate::createClickHandles()
 {
+    auto* qgsPage = dynamic_cast<QGSPage*>(scene());
```

Comment:
```suggestion
    auto* qgsPage = qobject_cast<QGSPage*>(scene());
```

Reply:
What does qobject_cast do for me that dynamic_cast does not?